code/modules/antagonists/gang/gang.dm

```diff
@@ -531,22 +407,373 @@
 	antag_hud_name = "Dutch"
+	gang_team_type = /datum/team/gang/dutch
 
-/datum/antagonist/gang/dutch/check_gang_objective()
-	for(var/D in get_antag_minds(/datum/antagonist/gang/dutch))
-		var/datum/mind/M = D
-		if(!considered_alive(M.current))
-			continue // dead people cant really do the objective lol
-		var/list/items_to_check = M.current.GetAllContents()
-		for(var/I in items_to_check)
-			var/obj/IT = I
-			if(istype(IT, /obj/item/stack/sheet/mineral/gold))
-				continue
-		return FALSE // didnt pass the bar check, no point in continuing to loop
-	var/obj/machinery/ore_silo/S = GLOB.ore_silo_default
-	var/datum/component/material_container/mat_container = S.GetComponent(/datum/component/material_container)
-	if(mat_container.materials[GET_MATERIAL_REF(/datum/material/gold)] >= 2000) // if theres at least 1 bar of gold left in the silo, they've failed to heist all of it
-		return FALSE
-	return TRUE
+/datum/team/gang/dutch/rename_gangster(datum/mind/gangster, original_name, starter_gangster)
+	var/static/regex/lasttname = new("\[^\\s-\]+$") //First word before whitespace or "-"
+	lasttname.Find(original_name)
+	if(starter_gangster)
+		gangster.current.fully_replace_character_name(gangster.current.real_name, "Head Cowboy [lasttname.match]")
+	else
+		gangster.current.fully_replace_character_name(gangster.current.real_name, original_name)
+	return
+
+
+/datum/antagonist/gang/irs
+	show_in_antagpanel = TRUE
+	name = "Internal Revenue Service Agent"
+	roundend_category = "IRS Agents"
+	gang_name = "Internal Revenue Service"
+	gang_id = "IRS"
+	acceptable_clothes = list(/obj/item/clothing/suit/irs,
+							/obj/item/clothing/under/costume/irs,
+							/obj/item/clothing/head/irs)
+	free_clothes = list(/obj/item/clothing/suit/irs,
+							/obj/item/clothing/under/costume/irs,
+							/obj/item/clothing/head/irs,
+						/obj/item/toy/crayon/spraycan)
+	antag_hud_name = "IRS"
+	gang_team_type = /datum/team/gang/irs
+
+/datum/team/gang/irs/rename_gangster(datum/mind/gangster, original_name, starter_gangster)
+	var/static/regex/lasttname = new("\[^\\s-\]+$") //First word before whitespace or "-"
+	lasttname.Find(original_name)
+	if(starter_gangster)
+		gangster.current.fully_replace_character_name(gangster.current.real_name, "Revenue Supervisor [lasttname.match]")
+	else
+		gangster.current.fully_replace_character_name(gangster.current.real_name, "Revenue Agent [lasttname.match]")
+	return
+
+/datum/antagonist/gang/osi
+	show_in_antagpanel = TRUE
+	name = "Office of Secret Intelligence Agent"
+	roundend_category = "O.S.I. Agents"
+	gang_name = "Office of Secret Intelligence"
+	gang_id = "OSI"
+	acceptable_clothes = list(/obj/item/clothing/suit/osi,
+							/obj/item/clothing/under/costume/osi,
```

Comment:
![image](https://user-images.githubusercontent.com/35135081/132933001-e74427d1-34df-4276-887a-9f9dee047f45.png)


Here and everywhere else

Reply:
You didn't fix this.

Reply:
Don't use tabs to align--indent up *once*.